src/Mvc/Mvc.Core/src/FromBodyAttribute.cs

```diff
@@ -10,9 +10,21 @@ namespace Microsoft.AspNetCore.Mvc;
 /// Specifies that a parameter or property should be bound using the request body.
 /// </summary>
 /// <remarks>
+/// Binds a parameter or property to the request body.
+///
 /// By default, ASP.NET Core MVC delegates the responsibility of reading the body to an input formatter.<br/>
 /// In the case of ASP.NET Core Minimal APIs, the body is deserialized by <see cref="System.Text.Json.JsonSerializer"/>.
+///
+/// For more information about parameter binding see
+/// <see href="https://learn.microsoft.com/aspnet/core/fundamentals/minimal-apis/parameter-binding?">Parameter binding</see>.
 /// </remarks>
+/// <example>
+/// In this example, the value of the 'name' field in the form-data request body is bound to the name parameter,
```

Comment:
It's a little hard to understand what is meant here when there is no `Person` type defined in the example code? Maybe we can add a record definition to make it clearer?